contracts/root/WithdrawManager.sol

```diff
@@ -147,7 +147,7 @@ contract WithdrawManager is DepositManager {
   }
 
   // withdraw tokens
-  function withdraw(
+  function withdrawBurntTokens(
```

Comment:
Questions and Assumptions:

Q. How long does it take until child-chain burnProof confirmed?
Asm. `~=3600sec`, until committed to rootchain

Q. How long does it take until `withdrawBurntTokens` got processed?
Asm. `Math.max256(_createdAt + 2 weeks, block.timestamp + 1 weeks)`

I'm bit missing spec. Plz feed me back when you are somehow free @jdkanani